tests/e2e/tests/test_notebook_images.py

```diff
@@ -0,0 +1,82 @@
+import subprocess
+import pytest
+
+from e2e.utils.constants import DEFAULT_USER_NAMESPACE
+from e2e.utils.config import metadata, configure_resource_fixture
+
+from e2e.conftest import (
+    region,
+)
+
+from e2e.fixtures.cluster import (
+    cluster,
+)
+from e2e.fixtures.kustomize import kustomize, clone_upstream
+from e2e.fixtures.profile_dependencies import (
+    configure_manifests,
+    profile_controller_policy,
+    profile_controller_service_account,
+    profile_trust_policy,
+    profile_role,
+    associate_oidc,
+    kustomize_path,
+)
+from e2e.fixtures.clients import (
+    account_id,
+    kfp_client,
+    port_forward,
+    session_cookie,
+    host,
+    login,
+    password,
+)
+from e2e.fixtures.notebook_dependencies import notebook_server
+
+TO_ROOT_PATH = "../../"
+CUSTOM_RESOURCE_TEMPLATES_FOLDER = "./resources/custom-resource-templates"
+
+NOTEBOOK_IMAGES = [
+    "public.ecr.aws/j1r0q0g6/notebooks/notebook-servers/jupyter-scipy:v1.5.0",
+    "public.ecr.aws/c9e4w0g3/notebook-servers/jupyter-tensorflow:2.6.3-gpu-py38-cu112-ubuntu20.04-v1.8",
+    "public.ecr.aws/c9e4w0g3/notebook-servers/jupyter-tensorflow:2.6.3-cpu-py38-ubuntu20.04-v1.8",
+    "public.ecr.aws/c9e4w0g3/notebook-servers/jupyter-pytorch:1.11.0-gpu-py38-cu115-ubuntu20.04-e3-v1.1",
+    "public.ecr.aws/c9e4w0g3/notebook-servers/jupyter-pytorch:1.11.0-cpu-py38-ubuntu20.04-e3-v1.1",
+]
+
+testdata = [
+    ("scipy", NOTEBOOK_IMAGES[0], "sanity_check.ipynb", "Hello World!"),
```

Comment:
naive question, why is the argument here "Hello World" instead of "1.5.0" it looks like the other inputs here follow the format of using the images version number

Reply:
It's not based on the image's name or tag but the output I am checking for is based on the python notebook that I am running. In case of scipy I am running the `sanity_check.ipynb` which simply prints Hello World. In the other 4 notebooks I check the version of the PyTorch or the TF framework since we want to make sure these frameworks are installed properly since these are images we built.